src/Enhavo/Bundle/UserBundle/Controller/ChangeEmailController.php

```diff
@@ -1,107 +0,0 @@
-<?php
```

Comment:
we should really avoid making it possible to change emails without confirmation/double-opt-in otherwise people can lock themselfes out or become unreachable.
so i will add the feature in a similar way of reset password